csunplugged/at_home/content/en/squeezing-pictures/inside-the-computer.md

```diff
@@ -0,0 +1,3 @@
+The small squares that you are using are the basis of most digital pictures - they are called “picture elements”, or “pixels” for short. A good quality photo or video can have millions of pixels in it; a million pixels is called a “megapixel”, and often the number of megapixels is used to indicate the quality of a camera.
+
+The idea of coding a run of the same colour with a number is called “run-length” coding. It can significantly reduce the amount of space needed to store the image on a computer, or the time needed to send it over a network. Run-length coding works particularly well for simple pictures that have only a few colours in them. It is the basis of sending black and white scans on fax machines, and in more recent times, variations of this idea are used in JPG files of photographs to save space. Run-length coding meant that fax transmissions would typically transmit 7 times faster than if every pixel was sent separately, because long runs of the same colour are common in high resolution scans. The general idea of looking for patterns in a file and replacing them with smaller codes is called data compression, and this is used to reduce the space that files take to store (such as photos and music on a mobile phone), and to make them faster to transmit (such as digital TV and online videos).
```

Comment:
```suggestion
The idea of coding a run of the same colour with a number is called “run-length” coding.
It can significantly reduce the amount of space needed to store the image on a computer, or the time needed to send it over a network.
Run-length coding works particularly well for simple pictures that have only a few colours in them.
It is the basis of sending black and white scans on fax machines, and in more recent times, variations of this idea are used in JPG files of photographs to save space.
Run-length coding meant that fax transmissions would typically transmit 7 times faster than if every pixel was sent separately, because long runs of the same colour are common in high resolution scans.
The general idea of looking for patterns in a file and replacing them with smaller codes is called data compression, and this is used to reduce the space that files take to store (such as photos and music on a mobile phone), and to make them faster to transmit (such as digital TV and online videos).
```